guake/callbacks.py

```diff
@@ -54,7 +54,7 @@ def on_search_on_web(self, *args):
             query = clipboard.wait_for_text()
             query = quote_plus(query)
             if query:
-                search_url = f"https://www.google.com/search?q={query}&safe=off"
+                search_url = "https://www.google.com/search?q={!s}&safe=off".format(query)
```

Comment:
What is this revert for? Have you not updated your repo from master yet?

Reply:
I have updated, and have cherry picked to use format, because the "f" seems to fail, is not quite working well. I have no idea, why are we using it, when it fails to compile

Reply:
f strings aren't supported in python 3.5 or below, but 3.5 is also very very EOL